platform-manageability-agent/go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/cenkalti/backoff/v4 v4.3.0
 	github.com/grpc-ecosystem/go-grpc-middleware/v2 v2.3.2
 	github.com/open-edge-platform/edge-node-agents/common v1.7.1
-	github.com/open-edge-platform/infra-external/dm-manager v0.2.2
+	github.com/open-edge-platform/infra-external/dm-manager v0.3.1-0.20250801100855-b22b6fb93b75
```

Comment:
Once https://github.com/open-edge-platform/infra-external/pull/147 is merged, this should be updated to point to the v0.3.1 tag that will be added for dm-manager